ivantiitsm_connector.py

```diff
@@ -146,8 +147,11 @@ def _suds_to_dict(self, sud_obj):
             return sud_obj.strftime(consts.HEAT_TIME_FORMAT)
 
         # Checking for NaN
-        elif sud_obj != sud_obj:
-            return None
+        try:
+            if math.isnan(float(sud_obj)):
+                return None
+        except:
+            self.debug_print('{} is not NaN'.format(sud_obj))
```

Comment:
The fix is also suggested by SemGrep: https://github.com/splunk-soar-connectors/ivantiitsm/runs/4860328725?check_suite_focus=true